src/components/CavaticaProjectList/CavaticaProjectItem.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {List, Icon, Button, Popup, Divider} from 'semantic-ui-react';
+import TimeAgo from 'react-timeago';
+
+const CavaticaProjectItem = ({projectNode, unlinkProject, studyId}) => {
+  return (
+    <List.Item>
+      <Icon
+        name={
+          projectNode.projectType === 'DEL'
+            ? 'paper plane outline'
+            : 'sliders horizontal'
+        }
+      />
+      <List.Content>
+        {unlinkProject && studyId ? (
```

Comment:
It would be nice to pass this in to the list on the Cavatica Project admin view, too.